domains/grid/components/GridWidgetText.vue

```diff
@@ -10,10 +10,8 @@ defineProps<Props>()
 
 <template>
   <div
-    :class="
-      'flex h-full flex-col items-center justify-center overflow-hidden rounded-[10px] p-4 text-neutral-100 ' +
-      bgColor
-    "
+    class="flex h-full flex-col items-center justify-center overflow-hidden rounded-[10px] p-4 text-neutral-100"
+    :style="{ backgroundColor: bgColor }"
```

Comment:
can you try using the cn  method from lukso/web-components instead?

Reply:
`cn` is just for merging tailwind classes, here we use `style` to apply color.